README.md

```diff
@@ -36,6 +36,11 @@ Usage of marathon_exporter:
   -marathon.uri string
         URI of Marathon (default "http://marathon.mesos:8080")
         Note: Supply HTTP Basic Auth (i.e. user:password@example.com)
+	            If you URI inclube '%' ,you should used marathon.username and marathon.password
```

Comment:
Typo fixes: "if your URI include '%', you should use...."

Also, about your PR, you have the issue when you have % in your password for instance right?

Reply:
I'm fix it.